test/jtreg-ext/requires/VMProps.java

```diff
@@ -462,7 +462,28 @@ protected String vmCDSForCustomLoaders() {
      * @return true if this VM can write Java heap objects into the CDS archive
      */
     protected String vmCDSCanWriteArchivedJavaHeap() {
-        return "" + ("true".equals(vmCDS()) && WB.canWriteJavaHeapArchive());
+        return "" + ("true".equals(vmCDS()) && WB.canWriteJavaHeapArchive()
+                     && isCDSRuntimeOptionsCompatible());
+    }
+
+    /**
+     * @return true if the VM options specified via the "test.cds.runtime.options"
+     * property is compatible with writing Java heap objects into the CDS archive
+     */
+    protected boolean isCDSRuntimeOptionsCompatible() {
```

Comment:
Please add braces for all the added `if` statements.

Reply:
Fixed.

Reply:
You missed this one:
```
        if (jtropts == null)
            return true;
```

Reply:
Fixed.